### DISCUSSION TITLE
[Application Development] Events not emitting from publicnode RPC

### DISCUSSION THREAD

**Discussion by TheMarstonConnell** [ethereum-optimism/developers#598] [Application Development]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

Been testing out a solidity contract that emits events. I have a listener set up to track these events written in golang. I've been using publicnodes websocket RPCs to do this and they work for Base & Sepolia but not OP testnet. Any ideas why this may be? Thanks.

**Comment by imtipi** [comment]
> not OP testnet

any logs?whats the output?
